plugin/pki/util.go

```diff
@@ -600,3 +604,14 @@ func shortDurationString(d time.Duration) string {
 	}
 	return s
 }
+
+func sha1sum(s string) string {
+	hash := sha1.New()
+	buffer := []byte(s)
+	hash.Write(buffer)
+	return hex.EncodeToString(hash.Sum(nil))
```

Comment:
Why passing nil as argument here? shouldnt the buffer be passed as argument?

Reply:
The buffer is already populated with the `hash.Write` function

Reply:
@rvelaVenafi , @EduardoVV already answered what I was going to say